### PR TITLE
Add wiki to tell how to run AAD integration test in localhost.

### PR DESCRIPTION
**Pull request by chenrujun**
Add wiki to tell how to run AAD integration test in localhost.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR.

File: sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+1. Create roles
+![role-user](images/role-user.png)
+
+2. Assign roles by clicking: "Azure Active Directory" -> "Enterprise Application" -> choose your application -> "Users and groups" -> assign the new roles to user1. (assignment of roles to groups is not available in the free tier of AAD).
+
+### Create groups
+1. Create 2 groups: group1 and group2. Add user1 in group1, and user2 in group2.
+Refs: [Create a basic group and add members using Azure Active Directory][Create a basic group and add members using Azure Active Directory]
+
+2. For user1, we need to create 111 groups: from group1 to group111. It's used in AADGroupsCountIT.
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md:82]: Can we make this optional? Community contributors will be scared to make contributions.

File: sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+3. In Edge, open an InPrivate window to change password for new created user.
+![open-inPrivate-window](images/open-inPrivate-window.png)
+![change-password](images/change-password.png)
+
+4. Now we get the value of `AAD_USER_NAME_1`, `AAD_USER_PASSWORD_1`.
+
+### `AAD_TENANT_ID_2`, `AAD_USER_NAME_2`, `AAD_USER_PASSWORD_2` 
+Get `AAD_TENANT_ID_2`, `AAD_USER_NAME_2`, `AAD_USER_PASSWORD_2` with the same method.
+
+### `AAD_MULTI_TENANT_CLIENT_ID`, `AAD_MULTI_TENANT_CLIENT_SECRET`.
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md:51]: And this multi-tenant application will be created in which tenant?

File: sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md:82]: 🤣

1. After make this optional, mvn command will fail with test failure.
2. In my understanding, no community contributors will run the test. Only member in our team will run it manually.

Originally, I have a plan to write script to create the 111 groups, but abandoned. Reasons:
1. Write script will cost more time than do it manually.
2. It's not a bad thing to let us be familiar with Azure Portal.

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md:82]: OK, I'll add `Optional`

File: sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md (hunk 2 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-test-aad/HOW_TO_RUN_AAD_INTEGRATION_TEST_IN_LOCALHOST.md:51]: Updated to
```
Register an application in the new created tenant (`aadIntegrationTest1`).
```

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**